meson.build

```
@@ -69,6 +69,7 @@ add_project_arguments ('-DGLIB_VERSION_MIN_REQUIRED=GLIB_VERSION_2_44', language
 
 # allow the daemon to include library files directly
 add_project_arguments ('-DPK_COMPILATION', language: 'c')
```

```suggestion
add_project_arguments ('-DPK_COMPILATION', language: ['c', 'cpp'])
```

This should work too and saves a line :-)